modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/CaseResult/CaseResultOutlet.tsx

```diff
@@ -118,19 +120,23 @@ const CaseResultOutlet = () => {
 	]);
 
 	useEffect(() => {
-		setTabs([
-			{
-				active: pathname === basePath,
-				path: basePath,
-				title: i18n.translate('result'),
-			},
-			{
-				active: pathname !== basePath,
-				path: `${basePath}/history`,
-				title: i18n.translate('history'),
-			},
-		]);
-	}, [basePath, pathname, setTabs]);
+		setTabs(
+			!isEditCase
+				? [
+						{
+							active: pathname === basePath,
+							path: basePath,
+							title: i18n.translate('result'),
+						},
+						{
+							active: pathname !== basePath,
+							path: `${basePath}/history`,
+							title: i18n.translate('history'),
+						},
+				  ]
+				: []
+		);
```

Comment:
You must not start validation by negating the value to be validated.

```suggestion
		setTabs(
			isEditCase
				? []
				: [
						{
							active: pathname === basePath,
							path: basePath,
							title: i18n.translate('result'),
						},
						{
							active: pathname !== basePath,
							path: `${basePath}/history`,
							title: i18n.translate('history'),
						},
				  ]
		);
```

Reply:
just upadet it!